src/App.js

```diff
@@ -1,17 +1,40 @@
-import React from 'react'
+import React, { useState } from 'react'
 import TripList from './components/TripList'
-import styled from 'styled-components'
+import styled from 'styled-components/macro'
+import CreateTrip from './components/CreateTrip'
+import TripsData from './data/tripsdata.json'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import Icons from './common/Icons'
+import ButtonAdd from './components/ButtonAdd'
 
 function App() {
+  const [trips, setTrips] = useState(TripsData)
```

Comment:
Maybee you can check before if there are trip datas and if not asign an empty array:
```
const tripsData = TripsData ? TripsData : []
const [trips, setTrips] = useState(TripsData)
````
Only an idea, no must.